src/libexpr/eval.cc

```diff
@@ -1513,13 +1513,13 @@ void EvalState::incrFunctionCall(ExprLambda * fun)
 }
 
 
-void EvalState::autoCallFunction(Bindings & args, Value & fun, Value & res)
+void EvalState::autoCallFunction(Bindings & args, Value & fun, Value & res, bool callFunctors)
 {
     auto pos = fun.determinePos(noPos);
 
     forceValue(fun, pos);
 
-    if (fun.type() == nAttrs) {
+    if (callFunctors && (fun.type() == nAttrs)) {
```

Comment:
It looks like this will prevent from calling _functors_ but not plain functions. Is it intended?